integration-tests/e2e/groups.cy.js

```diff
@@ -74,6 +76,30 @@ context('Groups', () => {
     groupDetails.childGroups().should('have.length', 1)
   })
 
+  it('should display group details and then allow to search users from link', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_OAUTH_USERS' }] })
+    cy.signIn()
+
+    cy.task('stubAssignableGroups', {})
+    cy.task('stubGroupDetailsWithChildren', {})
+    cy.visit('/manage-groups')
+    const groups = GroupsPage.verifyOnPage()
+    groups.groupRows().get('[data-qa="edit-button-Site 1 - Group 2"]').click()
+
+    const groupDetails = GroupDetailsPage.verifyOnPage('Site 1 - Group 2')
+    cy.task('stubAssignableGroups', { content: undefined })
+    cy.task('stubExtSearchableRoles', {})
+    cy.task('stubExternalUserSearch', {
+      content: replicateUser(1),
+      totalElements: 1,
+      page: 0,
+      size: 1,
+    })
+    groupDetails.searchUsers()
+
+    ExternalUserSearchPage.verifyOnPage()
+  })
```

Comment:
Do we need to check if the page is filtering by group code?

Reply:
that is cover by the normal search